modules/postgres/postgres_test.go

```diff
@@ -26,38 +27,49 @@ const (
 	password = "password"
 )
 
+var (
```

Comment:
How would you see moving this wait strategy to the module to be exposed to consumers?

Reply:
It's worked very well in my test, so sure!